### DISCUSSION TITLE
Get rid of a specific contributor

### DISCUSSION THREAD

**Discussion by SoftCreatR** [community/community#22424] [New to GitHub]
<p>Hi,</p>
<p>for reasons, I had to change the name and email address for all commits within my repository, that were made by a specific user*. This worked fine so far and all affected commits have been rewritten. I’ve also removed all releases/tags, wich referenced this specific user, so it doesn’t appear anywhere, except for the list of Contributors. Did I miss something, or isn’t there a way to get rid of this specific user (except for forcing him to delete his account ^^).</p>
<p>Btw, the user isn’t listed in the contributors overview (Insights) anymore. Just in the contributors box on the repositories index page.</p>
<p>* =</p>
<pre><code class="lang-shell">git filter-branch --env-filter '
WRONG_EMAIL="xxx"
NEW_NAME="yyy"
NEW_EMAIL="zzz"

if [ "$GIT_COMMITTER_EMAIL" = "$WRONG_EMAIL" ]
then
    export GIT_COMMITTER_NAME="$NEW_NAME"
    export GIT_COMMITTER_EMAIL="$NEW_EMAIL"
fi
if [ "$GIT_AUTHOR_EMAIL" = "$WRONG_EMAIL" ]
then
    export GIT_AUTHOR_NAME="$NEW_NAME"
    export GIT_AUTHOR_EMAIL="$NEW_EMAIL"
fi
' --tag-name-filter cat -- --branches --tags
</code></pre>

**Comment by canuckjacq** [comment, marked as answer]
<p>The contributors listing relies heavily on caching I think  so we may just need to trigger a rebuild. You can drop a link to the repository here if you want, or if it’s all private, feel free to <a href="https://support.github.com/contact">open a ticket</a>. You can even link to this thread to avoid typing it all out again :slight_smile:</p>

**Comment by SoftCreatR** [comment]
<p>Seems to be resolved now. Ty.</p>

**Comment by NyFrdo** [comment]
<p>Hi there , I met similar issue . I accidentally use a gitlab account to commit sth . to my github repo , can you help get rid of this account in my contributor list?Much thx</p>

**Comment by web-developer-django25** [comment]
Request to purge contributor cache for repository

I have a repository at https://github.com/web-developer-django25/contacts-app where I've rewritten the Git history to remove old contributor information. I've force pushed a new history with a single author, but the Pulse page still shows '2 authors'. Please help purge the contributor cache for this repository.
